src/Interface/Modules/Render/ES/SRCamera.cc

```diff
@@ -141,7 +141,9 @@ namespace SCIRun {
     //----------------------------------------------------------------------------------------------
     void SRCamera::setClippingPlanes()
     {
-      mRadius = (mSceneBBox.get_max() - mSceneBBox.get_min()).length() / 2.0f;
+      //we could divide by 2.0 for tightest bound but here we divide by 1.0 to accomidate
+      //for the clipping plane visualization
+      mRadius = (mSceneBBox.get_max() - mSceneBBox.get_min()).length() / 1.0f;
```

Comment:
What does the divide by one do? Simply convert to float without changing the value? To do that use a `static_cast<float>`--the divide by 1 just looks weird